app/assets/stylesheets/main.css.scss

```diff
@@ -283,6 +283,11 @@ ul {
   padding-top: lines(0.05);
 }
 
+.icon-remove {
+  top: -2px !important;
```

Comment:
Does this work without "!important". I would go to great lengths to avoid using that one.